### PR TITLE
Add addmv to decomposition list

### PR DESCRIPTION
**Pull request by Chillee**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #97542
* #97428
* __->__ #97427
* #97289
* #97293

cc @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @desertfire

### CODE REVIEW

File: torch/_inductor/decomposition.py [hunk 1]
```diff
@@ -18,6 +18,7 @@
 
 inductor_decompositions = get_decompositions(
     [
+        aten.addmv,
```
Comment (ngimel) [inline torch/_inductor/decomposition.py:21]: should it go to core decompositions?

**Review by ngimel — COMMENTED** [review]
What about perf? Last time you checked it it was disappointing compared to cublas?

**Comment by Chillee** [comment]
It's still slow unfortunately 🤔

**Comment by ngimel** [comment]
So if it's slow we shouldn't use the decomposition?

**Pull request closed without merging**